SMS19/settings.py

```diff
@@ -79,6 +81,11 @@
 # AUTH_USER_MODEL = 'main.UserProfile'  
 LOGIN_REDIRECT_URL = '/'
 
+ACCOUNT_SIGNUP_PASSWORD_ENTER_TWICE = False
+
+ACCOUNT_FORMS = {
+    'signup': 'main.forms.CustomSignupForm',
+}
```

Comment:
This custom form tells the AllAuth backend what all fields it is supposed to expect from the signup form . I had to do this as by default it does not expect name from signup form